### DISCUSSION TITLE
`migrate reset` resets DB and tries to seed without pushing schema

### DISCUSSION THREAD

**Discussion by nilaymaj** [prisma/prisma#8742] [Q&A] [answered]
I'm slightly new to SQL databases, especially new to migrations. So please point it out if there's something I'm not doing right.

I've written a seeding script that works fine when run separately using `prisma db seed`. Now, running `prisma migrate reset` seems to do the following:
1. delete all data from database, including the tables themselves
2. generate `@prisma/client`
3. run the seeding script

The problem is that since the command does not push the schema to the database (so tables are not recreated), my seeding script fails with a "table does not exist in current database" error. Ideally if this command automatically runs the seeding script, shouldn't it also push the schema to the db too? What am I missing here?

**Comment by ryands17** [comment, marked as answer]
@nilaymaj 👋 

This seems like a bug. It would be great if you open an issue [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with the reproduction steps so that we can look into this.

**Reply by nilaymaj** [reply]
Thanks, have created issue #8766

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
